src/System.Diagnostics.Process/src/System/Diagnostics/ProcessStartInfo.WinRT.cs

```diff
@@ -0,0 +1,19 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Diagnostics
+{
+    public sealed partial class ProcessStartInfo
+    {
+        public string[] Verbs => Array.Empty<string>();
```

Comment:
I didn't finish scouring over the existing code (which this is)- we need a lot more tests to fill in the holes.